uk.co.compucorp.civicrm.hrleaveandabsences/templates/CRM/HRLeaveAndAbsences/Page/AbsenceType.tpl

```diff
@@ -1,5 +1,8 @@
-{if $action eq 1 or $action eq 2 or $action eq 8}
-  {include file="CRM/HRLeaveAndAbsences/Form/AbsenceType.tpl"}
+{if $action eq 1 or $action eq 2}
```

Comment:
Can we use some other notation (like here https://www.smarty.net/forums/viewtopic.php?p=34638) so we check is the action `IN` one of 2 options instead of checking if it equals one and then another?

Reply:
Yes we can. This is good especially if we have many options 🤷‍♂️